src/app/game/tic-tac-toe/multi-player/page.js

```
@@ -1,254 +1,390 @@
+        </div>
+      )}
+
+      {mySymbol && (
+        <div className={styles.playerRole}>
+          You are <b>{mySymbol}</b>
+        </div>
+      )}
+
+      {/* WAITING: Show if game started but both players not joined */}
+      {gameNumber && !bothPlayersJoined && (
+        <output className={styles.statusBar} role="status" aria-live="polite">Waiting for opponent to join...</output>
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)